package.md

```diff
@@ -48,8 +48,6 @@ This will test the current state of CSS and JS code. Tests **MUST** be executed
   > core
```

Comment:
Not urgent, but you probably want to update this file. (I myself can live without this file, as you know ;-) )